index.js

```diff
@@ -11,11 +11,16 @@ class TooManyFilesError extends Error {
 	}
 };
 
-module.exports = (rootDirs, delay = 50) => {
+module.exports = (rootDirs, { delay = 50, suppressReporting }) => {
```

Comment:
I think that makes sense, but why not reverse the option? Make it a `reporting` option that defaults to `false` – then it's also not a breaking change.

Reply:
I figured it's a breaking change either way because the second argument is now an options object.

I opted for `suppressReporting` to make it abundantly clear that this would be changing the (new) default behavior - seemed more explicit than `report: false`, plus defaulting to `true` on a boolean argument always struck me as a little weird/unexpected. But I don't care too much either way.

More generally (cf. #7), I'm not worried about breaking changes (i.e. releasing a new minor version) at this point: Updating faucet-pipeline should be a breeze, I doubt there are any other users anyway.

Reply:
Ok, let's ship it 👍